testing/openresty/APKBUILD

```diff
@@ -139,7 +140,7 @@ _module() {
 	local _deps="$(eval echo "\$_module_${name//-/_}_deps")"
 
 	pkgdesc="$pkgdesc (module $soname)"
-	depends="cmd:nginx${_deps:+ }${_deps}"
+	depends="${_provide}${_deps:+ }${_deps}"
```

Comment:
will `openresty` (1.15) modules work with `nginx` (1.16) and vice versa?

Wouldn't it be better to depend on the exact nginx implementation?